### DISCUSSION TITLE
02enKF_1DinverseHeatTransfer

### DISCUSSION THREAD

**Discussion by huaji2005** [ITHACA-FV/ITHACA-FV#493] [General]
Hello,

I try to compile the tutorial 02enKF_1DinverseHeatTransfer on UQ directory, but I got an error message: fatal error: MUQ/Modeling/Distributions/Gaussian.h: No such file or directory. Could you please let me know where I can find it. Thanks.

John

**Comment by umbertoEmil** [comment]
Hi John,

The possible issues are two: either you did not install MUQ or you did not make the export to the MUQ library.
Please, check if you added to your bashrc the exports
```
export MUQ_LIBRARIES="/directoryWhereYouInstalledMUQ" #$MUQ_LIBRARIES"
export MUQ_EXT_LIBRARIES="/directoryWhereYouInstalledMUQ/muq_external" #$MUQ_EXT_LIBRARIES"

```
Let me know if this solved the problem.

Umberto

**Comment by huaji2005** [comment]
Hello Dr. Morelli,

I am wondering if ITHACA-FV can solve an inverse heat conduction problem
with two unknown boundary heat fluxes. If yes, could you please give me
some instructions?

Thanks a lot for your help,

John

**Reply by umbertoEmil** [reply]
Hello John,

Sorry for the late reply. The procedure to solve such a problem should be very similar to the one that we already implemented in ITHACA. However, I expect the problem to be much more ill conditioned and, as such, could require some intensive regularization. Did you have the chance to make some tries during this time?

Best regards,
Umberto

**Comment by huaji2005** [comment]
Hello Dr. Morelli,

I have not tried yet. I am not quite sure how to do that. Should I use two
parameterizations for the two unknown boundary heat fluxes?

Thanks a lot,

John
